core/src/main/java/jenkins/widgets/HistoryPageFilter.java

```diff
@@ -101,85 +105,100 @@ public void setSearchString(@Nonnull String searchString) {
     /**
      * Add build items to the History page.
      *
-     * @param items The items to be added. Assumes the list of items are in descending queue ID order i.e. newest first.
+     * @param runItems The items to be added. Assumes the list of items are in descending queue ID order i.e. newest first.
      */
-    public void add(@Nonnull List<T> items) {
-        if (items.isEmpty()) {
+    public void add(@Nonnull Iterable<T> runItems) {
```

Comment:
I don't know if this was intended to really be a public method for consumption in plugins, but maybe @tfennelly could shed some light as he wrote the original code.

Is there a good way to tell if this has been used by plugins?